ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml

```diff
@@ -2113,6 +2113,7 @@ tests:
   steps:
     cluster_profile: ibmcloud-qe
     env:
+      ENABLE_REBOOT_CHECK: "true"
```

Comment:
Suggest to run the test case in `openshift-verification-tests-master__installation-nightly-4.1N.yaml` like other platforms.